recipe/meta.yaml

```
@@ -1,45 +1,65 @@
-{% set name = "<PLEASE ADD PKG NAME>" %}
-{% set version = "<PLEASE ADD PKG VERSION>" %}
+{% set name = "pythonanywhere-core" %}
+{% set version = "0.2.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: <sha256>
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pythonanywhere_core-{{ version }}.tar.gz
+    sha256: 2b43d726c18e5972a01bdfe09482c72dad20325fc3196c7504f4d90acaf642c0
+
+  # tests and tests data files
+  - url: https://github.com/pythonanywhere/pythonanywhere-core/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: b338361925ed92ef4e3ff33b29fa4c05f003a7fa470b0157a2e427e626bb4202
+    folder: gh_src
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  skip: True  # [py<38]
+  skip: True  # [py<310]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - python
+    - poetry-core
     - pip
   run:
     - python
+    - python-dateutil >=2.8.2
```

https://github.com/pythonanywhere/pythonanywhere-core/blob/v0.2.5/pyproject.toml#L22
```suggestion
    - python-dateutil >=2.8.2,<3.0
```